tests/integration/components/post-new-form-test.js

```diff
@@ -64,3 +67,16 @@ test('it has a loading indicator when previewing and post is saving', function(a
 
   assert.equal(this.$('.editor-with-preview .spinner').length, 1);
 });
+
+test('it renders option to create task or progress if user is at least contributor in organization', function(assert) {
+  assert.expect(2);
+
+  this.register('service:credentials', Ember.Service.extend({
+    currentUserMembership: { isContributor: true }
+  }));
+
+  this.render(hbs`{{post-new-form post=post placeholder=placeholder}}`);
+
+  assert.equal(this.$('option[value=task]').length, 1, 'task option is rendered');
+  assert.equal(this.$('option[value=progress]').length, 1, 'progress option is rendered');
+});
```

Comment:
I don't think we're testing enough here. I think we need an explicit test for them not being a contributor. I would also test that the contributor has the other options still.